From bc86d603e632ce61de4ae7a5d4bcef080cbdb160 Mon Sep 17 00:00:00 2001
From: Bartosz Golaszewski <brgl@bgdev.pl>
Date: Wed, 11 May 2022 13:45:06 +0200
Subject: [PATCH] setup: don't use setuptools-markdown

This project is deprecated and irrelevant for the functionality of
eth-utils. We don't support it in meta-python so just drop it from
the dependencies.

Signed-off-by: Bartosz Golaszewski <brgl@bgdev.pl>
---
Upstream-Status: Pending

 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 5717d5b..cb62132 100644
--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,6 @@ setup(
     # *IMPORTANT*: Don't manually change the version here. Use the 'bumpversion' utility.
     version='1.1.2',
     description="""Common utility functions for ethereum codebases.""",
-    long_description_markdown_filename='README.md',
     author='Piper Merriam',
     author_email='pipermerriam@gmail.com',
     url='https://github.com/ethereum/eth_utils',
@@ -48,7 +47,6 @@ setup(
         "toolz>0.8.2,<1;implementation_name=='pypy'",
         "cytoolz>=0.8.2,<1.0.0;implementation_name=='cpython'",
     ],
-    setup_requires=['setuptools-markdown'],
     extras_require=extras_require,
     py_modules=['eth_utils'],
     license="MIT",
-- 
2.34.1

